### DISCUSSION TITLE
Odd behaviour when Eddi restarted and looking at "green energy today"

### DISCUSSION THREAD

**Discussion by DBT85** [CJNE/ha-myenergi#354] [General]
Today I noticed that one of my Eddi's had lost its internet connection. I went round and restarted it, noting that it had diverted 14kwh today. Once restarted I expected to see the sensor.myenergi_eddi_hub_green_energy_today populate with that same 14kwh number, however it said 0 while at the same time the givenergy app on my phone proudly showed the 14kwh number and the consumption history graph looked somewhat strange at the time, it has now (2 hours later) resolved to appear like a proper chart.

Anyway, I thought it odd that the green energy number hadn't pulled through properly. Looking again now 2 hours later it has started to increase its value, but only gradually. Almost like it is re-living the day as it happened several hours ago.
![Captursdsdfsdve](https://user-images.githubusercontent.com/10498438/230690219-50c8ccd8-3301-46b2-9540-7f16abd7aa82.JPG)

As it has now rolled back around to midnight, the final number never did get a chance to grow beyond 3.16kwh and display the true 14kwh figure. We are back at 0 again and it has not continued to climb from the previous days tally. I just wondered if anyone else had noticed it? If I get a chance next time it loses connection I'll reboot it a bit earlier and keep an eye on it.

**Comment by ats57** [comment]
I noticed that many entyties has dropped out since latest HA Core update.
![image](https://github.com/CJNE/ha-myenergi/assets/116253395/3039d0cb-14db-41e2-ac92-3a43c431596b)
